### ISSUE TITLE
Fine-tuning the model on a custom dataset?

### ISSUE THREAD

**Issue by ImSo3K** [google-research/simclr#194]
I have a video dataset that is already sliced into frames, where around ~80% of it is unlabelled (training data) and the rest is labeled (test data) where each range of frames has a label.
I searched the Issues section and found [this](https://github.com/google-research/simclr/issues/50#issuecomment-652117563) comment suggesting to create a [tf dataset](https://www.tensorflow.org/datasets/add_dataset), my two questions are:

1. Is there an easier way?

2. If not, I did try to build my dataset but some things seem strange to me:

here is my `_info` method:

```
  def _info(self) -> tfds.core.DatasetInfo:
    """Returns the dataset metadata."""
    # TODO(DAD_dataset): Specifies the tfds.core.DatasetInfo object
    return tfds.core.DatasetInfo(
        builder=self,
        description=_DESCRIPTION,
        features=tfds.features.FeaturesDict({
            # These are the features of your dataset like images, labels ...
            'image': tfds.features.Video(shape=(None, 224, 224, 1), encoding_format='png', dtype=tf.uint8),
            'label': tfds.features.ClassLabel(names=['anomalous', 'normal']),
        }),
        # If there's a common (input, target) tuple from the
        # features, specify them here. They'll be used if
        # `as_supervised=True` in `builder.as_dataset`.
        supervised_keys=None,  # Set to `None` to disable
        disable_shuffling=False,
    )
```

I guess it's better to use `tfds.features.Video` for a sequence of frames, but does [`disable_shuffling`](https://www.tensorflow.org/datasets/add_dataset#maintain_dataset_order) shuffles the order of the frames for each video or just shuffles between the order of each batch of frames(video)?

Now they demonstrate [`split_generators`](https://www.tensorflow.org/datasets/add_dataset#_split_generators_downloads_and_splits_data) with downloading the data, but my data exists locally on my machine, how can I read the data from my actual PC?
